chart/values.yaml

```diff
@@ -15,12 +15,10 @@ timescaledb-single:
   # disable the chart if an existing TimescaleDB instance is used
   enabled: &dbEnabled true
   
-  # TODO(paulfantom): switch to official image before 0.10.0 release
+  # override default helm chart image to use one with newer promscale_extension
   image:
-    #repository: timescale/timescaledb-ha
-    #tag: pg14.2-ts2.6.1-p3
-    repository: ghcr.io/timescale/dev_promscale_extension
-    tag: 0.5.0-alpha-ts2-pg14
+    repository: timescale/timescaledb-ha
+    tag: pg14.2-ts2.6.1-p4
```

Comment:
for next release: should consider tags like `pg14.2-ts2.6.1-latest`. Don't know

Reply:
There is an open issue to figure this out across the board - https://github.com/timescale/tobs/issues/348